### PR TITLE
Better UX For BoltConfiguration

### PR DESCRIPTION
**Pull request by Naokimi**
Asking users to copy-paste a url is bad UX, instead we should ask them to decide between test and production environment and handle the related urls with the backend logic.

This will become a greater issue down the line as the url to display iframes is different from the one to making API calls, meaning that without this change we would ask for even more copy-pasted urls.

### CODE REVIEW

File: app/controllers/spree/admin/bolts_controller.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def bolt_configuration_params
           .require(:solidus_bolt_bolt_configuration)
           .permit(
             :bearer_token,
-            :environment_url,
+            :test,
```
Comment (MassimilianoLattanzio) [inline app/controllers/spree/admin/bolts_controller.rb:33]: The `test` name sounds a little bit generic to me and can be misleading. WDYT about using a more specific name?

Comment (Naokimi) [reply app/controllers/spree/admin/bolts_controller.rb:33]: what do you think of `sandbox`?

Comment (MassimilianoLattanzio) [reply app/controllers/spree/admin/bolts_controller.rb:33]: Yeah I think it's better

**Review by DanielePalombo — CHANGES_REQUESTED** [review]
Bolt offers 3 environments:

https://api.bolt.com
https://api-sandbox.bolt.com
https://api-staging.bolt.com

I think we should provide a select input to select one of them

File: spec/models/solidus_bolt/bolt_configuration_spec.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
     [
       'id',
       'bearer_token',
-      'environment_url',
+      'test',
```
Comment (MassimilianoLattanzio) [inline spec/models/solidus_bolt/bolt_configuration_spec.rb:8]: I think you need to change also this, right?

**Review by DanielePalombo — APPROVED** [review]

**Review by MassimilianoLattanzio — APPROVED** [review]
LGTM

**Pull request merged**